core/server/services/auth/session/index.js

```diff
@@ -0,0 +1,130 @@
+const {URL} = require('url'),
+    common = require('../../../lib/common'),
+    config = require('../../../config'),
+    settingsCache = require('../../settings/cache'),
+    {User} = require('../../../models/user'),
+    {Session} = require('../../../models/session'),
```

Comment:
I am not a huge fan of these fn extractions and a big fan of package notation.

e.g. if i want to search for `models.Session` usages in the code base, i won't find this usage. 

The code is IMO much harder to read.

> store: new SessionStore(Session),

Reply:
> {BadRequestError, UnauthorizedError, InternalServerError} = require('../../../lib/common/errors');

Same here.

The whole codebase uses

`new common.errors.BadRequestError`

Two general advantages:
- if you read the code, you know where the impl is coming from (i don't have to look on the top and search where the import is coming from)
- it's consistent with the code base

Reply:
Good points, agreed :smile_cat: